crossing-training.lic

```diff
@@ -301,6 +302,28 @@ class CrossingTraining
       fput('cast')
     end
 
+    if data['moon']
+      moon = UserVars.moons['visible'].first
+      unless moon
+        echo "No moon available to cast #{data['abbrev']}"
+        unless Script.running? 'moonwatch'
+          echo "moonwatch is not running. Starting it now"
+          custom_require 'moonwatch'
+          echo "Run `;autostart add moonwatch` to avoid this in the future"
+        end
+        weather = bput('weather', "inside", "You glance up at the sky.")
```

Comment:
I found out `weather` is a way to check if you're inside without suffering round time.